idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -291,34 +297,38 @@ def _make_params(self):
         _total_active_plate_number = 2 * sum(self.config.channel_list) - 1 -\
             self.config.divider_plate_number
 
-        self.plate_area = Expression(expr=self.config.total_area /
+        self.plate_area = Expression(expr=self.config.total_area*pyunits.m**2 /
```

Comment:
This looks like a very bad idea. Why is the config argument itself appearing in the Expression - this looks like it should just be a unit level `Var`.